src/pages/about.astro

```diff
@@ -103,12 +103,37 @@ const members = filterInactiveMembers(await getMembers());
       <section>
         <h2>What payments are eligible?</h2>
 
-        <p
-          >Companies directly pay maintainers of Open Source projects they depend on, with no strings attached. Paying
-          developers to do specific work does not count towards the Pledge. Feel free to participate in <a
-          href="https://github.com/opensourcepledge/osspledge.com/issues/112">our ongoing discussion</a> on what counts
-          and what doesn't.</p
-        >
+        <p>The primary intent with the Pledge is <b>no-strings-attached payments
+        to upstream maintainers</b>. Start there.</p>
+
+        <ol>
+
+          <li>Open Source projects that your company controls are out of scope.
+            If you own the trademark, it doesn't count.</li>
+
+          <li>Open Source projects that exclusively benefit your company or
+            your company's own ecosystem are out of scope.</li>
+
+          <li>It's okay if you get non-exclusive logo placement or other
+            acknowledgement for your payments, or an hour or two of consulting
+            time with the project team, but anything beyond such minimal
+            benefits puts the payment out of scope.</li>
+
+          <li>Payments to foundations are generally in scope, though most
+            foundations today are not set up to pay maintainers (they focus
+            more on conferences, trademarks, etc.). We hope that the Pledge
+            will enable foundations to shift towards paying maintainers in the
+            future.</li>
+
+          <li>Platforms such as thanks.dev, Open Collective, and GitHub
+            Sponsors are a common way to pay maintainers, though they can also
+            be used to make payments that are out-of-scope for the Pledge.</li>
```

Comment:
Maybe it's worth clarifying that there are two separate things here. When using these platforms, there's the question of

1. _who_ is being paid — only recipients that fit the above criteria are eligible; and
2. _how much_ of the payment the recipient is receiving — minimal fees can be ignored, but the vast majority of money should go to maintainers.

For example, if I make a payment using thanks.dev, but tip thanks.dev 50% of the payment, I shouldn't be able to count the full payment.

Reply:
> if I make a payment using thanks.dev, but tip thanks.dev 50% of the payment, I shouldn't be able to count the full payment

A tricky part of this is that Open Collective (and GitHub Sponsors) are not as easy to quantify this for – Open Collective has one fee for the platform and then eg. the Open Source Collective has one fee for money flowing into the project

There's also often fees when withdrawing money, especially if one withdraws eg. Thanks.dev or GitHub Sponsors to the project's Open Collective.

Treating thanks.dev differently because it allows for a custom platform fee doesn't feel fair – especially as its the most transparent and simple approach and one I think should be encouraged rather than discouraged

Reply:
Our goal for the Pledge over the next year should be to surface all of this sort of nuance, to bring clarity to the kinds of institutions in the ecosystem (companies, platforms, projects) and establish norms for transparency and accountability between all parties. That will be an ongoing conversation. Our goal right now is to get the conversation started.